kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -419,9 +434,9 @@
       },
       numBarActions() {
         let maxSize = 1;
-        if (this.windowBreakpoint === 1) {
+        if (this.windowBreakpoint === 1 || this.windowBreakpoint === 2) {
```

Comment:
This if statement could instead check if `this.windowBreakpoint` is less than or equal to 2 as a small refactor.

Reply:
Ah that will not work as when the size of breakpoint will go less than 1 we want to show only one icon the refactor you are saying will make two icon to show in that case but originally intended was 1 .

we can change it to get value greater than equal to 1 or less than equal to 2 but i think that will not be of any use.

Reply:
Okay, I understand it now, I overlooked that detail before :)